src/vision/test/hsv-filter-test.cpp

```diff
@@ -0,0 +1,43 @@
+/*
+ * Created By: Reid Oliveira
+ * Created On: January 06, 2018
+ * Description:
+ */
+
+#include "./ImageTestUtils.h"
```

Comment:
Paths should not be relative, we should let the compiler find headers for us. ie: `#include "ImageTestUtils.h"`

Reply:
done